src/solver/simulation/include/antares/solver/simulation/solver.hxx

```diff
@@ -906,6 +899,7 @@ void ISimulation<ImplementationType>::computeAnnualCostsStatistics(
             pAnnualStatistics.updateTime.addCost(s.averageUpdateTime);
         }
     }
+    durationCollector.summary();
```

Comment:
Here we make the assumption that no other simulation is run in the same process:
we don't want to apply this limitation on the library.

For example, at a later point we could be willing to run multiple simulations from python or any other client libs.
This line will print, for simulation 1, the times of all other running simulations.

Reply:
Use of summary here is for test purposes, the recorded times will be written in a file